llmebench/models/HuggingFace.py

```diff
@@ -0,0 +1,48 @@
+import logging
+import json
+import requests
+from llmebench.models.model_base import ModelBase
+import os
+
+from tenacity import (
+    retry,
+    retry_if_exception_type,
+    stop_after_attempt,
+    wait_random_exponential,
+)
+
+API_TOKEN = os.environ["HUGGINGFACE_API_TOKEN"]
```

Comment:
Lets make this an argument of the class constructor, along with the default env var value (to match other models)

Reply:
:+1: